core/templates/dev/head/css/oppia.css

```diff
@@ -952,6 +952,10 @@ textarea {
   color: rgba(255,255,255,1);
 }
 
+.btn:focus, .btn:active:focus {
+    outline: 0px;
```

Comment:
can you just use "0"? We omit units for "0" values.

More importantly: what is actually causing the outline? Is it a default browser thing, or is it something we've coded? If the latter, we should remove what we've done rather than overwrite it with this.